### ISSUE TITLE
Play but skip beginning/end

### ISSUE THREAD

**Issue by isedwards** [starplayerpc/starplayerpc#5]
An option to right click on the track and prevent the playing of the first and/or last 1 second (useful if these contain a fade in/out, or a data fade that are not desired).

Requires a menu to be displayed when right-clicking.

**Comment by Steve-Palmer** [comment]
See #4 - could be generalised by setting a start and end time for playing the clip. To achieve this, the clip would start playing at 1 second and end at n-1 (where n is the duration in seconds of the clip).

**Comment by isedwards** [comment]
I've implemented start-time and stop-time.  Unfortunately, VLC does not support negative times relative to the end of the video (this would have made things really easy).  I've looked at methods retrieve the video duration within .NET but there's nothing great available out of the box, the [file's metadata](https://stackoverflow.com/a/18819029/1624894) only stores duration to the [nearest second](https://stackoverflow.com/questions/10190906/how-to-get-video-duration-from-mp4-wmv-flv-mov-videos#comment70643361_18819029) which could lead to problems.

I suspect that allowing negative/relative end times could be easily fixed with `libvlc` as we'd have access to all the file's exact metadata for all the media types that VLC supports at run-time.

For now, the user is able to leave the end time blank to play to the end of the file, or else they need to find out the duration and enter the end time they want in seconds.
